tests/parse_datetime_test.py

```diff
@@ -497,7 +497,7 @@ def test_description():
          'DTSTART;TZID=Europe/Berlin;VALUE=DATE-TIME:20140216T080000',
          'DTEND;TZID=Europe/Berlin;VALUE=DATE-TIME:20140216T090000',
          'DESCRIPTION:please describe the event',
-         'RRULE:FREQ=DAILY;UNTIL=20150605T000000')))
+         'RRULE:FREQ=DAILY;UNTIL=20150604T230000Z')))
```

Comment:
Do you think it makes sense to create a copy of this test with non-localised times?

Reply:
Done.